recipes/write_behind/README.md

```diff
@@ -0,0 +1,20 @@
+# RedisGears Write-Behind Recipe
+
+## Deployment options
+
+* [With Redis OSS](oss/README.md)
+* [With Redis Enterprise Software and Oracle](oracle/rs/README.md)
+* [With Redis Enterprise Software and MySQL](mysql/rs/README.md)
```

Comment:
Still think we should drop the enterprise deployment doc for now cause this is not how its going to be in the end. We can save it as internal doc for internal usage ..